packages/api-v4/src/account/types.ts

```diff
@@ -604,8 +604,8 @@ export interface AccountBeta {
   label: string;
   started: string;
   id: string;
-  ended?: string;
-  description?: string;
+  ended?: string | null;
+  description?: string | null;
```

Comment:
Should the `?`s be removed? I think `ended` and `description` will always be in the response

```suggestion
  ended: string | null;
  description: string | null;
```

Reply:
yup! done